client/state/plans/selectors.js

```diff
@@ -43,7 +48,7 @@ export const getPlan = createSelector(
  */
 export function getPlanRawPrice( state, productId, isMonthly = false ) {
 	const plan = getPlan( state, productId );
-	if ( ! plan || ! ( plan.raw_price || plan.raw_price === 0 ) ) {
+	if ( get( plan, 'raw_price', -1 ) < 0 ) {
```

Comment:
I see. My point is that a programmer who didn't read the lodash docs or is not super familiar with `get` won't be able to parse this line quickly. Or to phrase it differently: I did not understand it at first sight, only after going through the lodash docs. :)